test/fixtures/cookbooks/rackspace_apache_php_test/attributes/fastcgi.rb

```diff
@@ -0,0 +1 @@
+default['apache']['mod_fastcgi']['download_url'] = 'http://d150860c924442f0c4ba-bb788a6a02be77ad65ac0686e629b047.r71.cf3.rackcdn.com/packages/fastcgi.tar.gz'
```

Comment:
This one is related to https://github.com/svanzoest-cookbooks/apache2/issues/415 . I'd suggest to set this attribute in the cookbook itself, and make sure the file is hosted in a maintained location